source/tools.txt

```diff
@@ -119,6 +119,10 @@ Flask
 Other Tools
 ~~~~~~~~~~~
 
+- `Full Stack FastAPI App Generator <https://github.com/mongodb-labs/full-stack-fastapi-mongodb>`__
+  is an app generator that enables you to quickly spin up a full-stack application
+  integrating FastAPI, React, and MongoDB.
```

Comment:
```suggestion
  that integrates FastAPI, React, and MongoDB.
```
S: Generally avoid using -ing verbs for clarity reasons. Use active voice instead.
Style Guide: https://www.mongodb.com/docs/meta/style-guide/writing/clarify-gerunds-participles/